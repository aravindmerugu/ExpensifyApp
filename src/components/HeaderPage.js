import React from 'react';
import { NavLink } from 'react-router-dom';

const HeaderPage = () => (
    <header>
    <h1>Expensify</h1>
    <ul>
    <li><NavLink end to="/" className={({ isActive }) => isActive ? 'is-active' : undefined }>Dashboard</NavLink></li>
    <li><NavLink end to="/create" className={({ isActive }) => isActive ? 'is-active' : undefined }>createpage</NavLink></li>
    <li><NavLink end to="/help" className={({ isActive }) => isActive ? 'is-active' : undefined }>help</NavLink></li>
    </ul>
    </header>
  )

export default HeaderPage