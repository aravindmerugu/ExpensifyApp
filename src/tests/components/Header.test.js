import React from 'react'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'
import HeaderPage from '../../components/HeaderPage'

test('should render Header correctly', () => {
    const wrapper = shallow(<HeaderPage/>)
    expect(toJSON(wrapper)).toMatchSnapshot()
})

