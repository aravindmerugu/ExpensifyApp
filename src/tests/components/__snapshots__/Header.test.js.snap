// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <ul>
    <li>
      <NavLink
        className={[Function]}
        end={true}
        to="/"
      >
        Dashboard
      </NavLink>
    </li>
    <li>
      <NavLink
        className={[Function]}
        end={true}
        to="/create"
      >
        createpage
      </NavLink>
    </li>
    <li>
      <NavLink
        className={[Function]}
        end={true}
        to="/help"
      >
        help
      </NavLink>
    </li>
  </ul>
</header>
`;
